### PR TITLE
Fix issue where vCard contact image would not display correctly

### PR DESCRIPTION
**Pull request by maxxcrawford**
Preview of the image: 

relay-logo

### CODE REVIEW

**Comment by Vinnl** [comment]
I think this fixes #2203?

File: api/renderers.py [hunk 1]
```diff
@@ -13,8 +13,8 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
...
...+K5vituw3aM+gOrsSabyxkCOy/A6QMEerIjUjR6943QcTmw4ikFqVPTYtHTqMO2h2HvY5ZN5ST64aYRWX+FM0efIvnEOSGiaM8d5pJ2yCdg1nuSGg76NP08bh+nVKgqLoLe2xRHxXKn5pt/KgML0j7m6gRsuR/G90BcTQMvcLIcXvmFkWSNu4XnHgFrnaOzQ7adISAekHHX58ml49lFqhgTdMHc2yG/AHj5FH7jT3VgYdo4vBN2fAcqhyGagrDo2AfTHoOq4Nivx5zsuwm2cOMwHCfNdFkC4S9wH6fuQy8shs5zoe1sha1T/rQEsIxnrMHuR2D8bRjZogIvnjeDD9j5CbhAAAs7FigmDGe21MgJq2nnE5HKlzYlVNFYGksV5kHH+dC5AYL2FnmzWwlY+kQN2P8s7HkYaiNQ6FJNqpgngqU6gSikJ+Dwdh008VukJ4cR1sr8eFVVErvOg/bzoXRaS73NrQcsfSaH4Ohm2PkghDkI8qlNYk7HDOlTiRxmWtVGQ4Ud66a8Q7oIWbrDV0Sqftx+JvReAW1nnIJJXwasnxx4CaiMwFv3wtgulTZyuzkxIOIwzK/eoG4SI304zDo+rCcmQXqSJSx8P5RXQcue1gWWPtURqI7B1nugMZH2oHJSabYBOFj1EQM12w6PtKcJ0bQzrcG826DnYjJLnQGrZU/lCEwMwr5HoXYktTqh07ExmsQWUrqdxvR4aUrCbkdPXGppCSx+f/rjQau/pRmwSNpYhX3fg7FtUDmoKAnO3XIvLCfTNmLt+xQ9FkLnWTDvBgg7VbKZnQxYx3OOozD4GAy/BvVRyLdbKjVtMuaCBF5JLFVRjS7d50LfFcpWZZDKgPW/i+slHPgRVA/DsVdVnThXtrGULgcJ1VfYGFZWav610HE6BMXsPcuA9X98RASHX1DwGt8KU2+b4D3J+PKd0Hs+9JwDbYsg3529Txmw/mXwasDkXgjb06lGgFy7yv4SExWetE3DGbCycxIfnr0F2cmAlZ0MWNnJgJWd7GTAyk4GrOxkwMpOdjJgZScDVnYyYGUnOxmwspMBKzsZsLKTnQxY2cmAlZ0MWNnJTgas7GTAyk4GrOxkJwNWdjJgZScDVnaykwErOxmwspMBKzvZyYCVnQxY2cmAlZ3sZMDKTgas7GTAyk52/mXnfwkwAHkfPlEQWHwkAAAAAElFTkSuQmCC"
```
Comment (maxxcrawford) [inline api/renderers.py:16]: question: Should I also add the un-encoded image somewhere?

Comment (groovecoder) [reply api/renderers.py:16]: Good idea. And maybe throw a comment in here to direct people where this came from.

Comment (maxxcrawford) [reply api/renderers.py:16]: Will do.

Comment (maxxcrawford) [reply api/renderers.py:17]: Added in ab6879f88888b404668fefd6bd10978c21a83f6c

**Review by groovecoder — APPROVED** [review]

**Pull request merged**